src/pkg/user/service/service.go

```diff
@@ -22,77 +28,247 @@ type Service struct {
 	Proc  *processing.Proc
 }
 
+type TokenMetadata struct {
+	UUID   string
+	UserID string
+}
+
+type TokenDetails struct {
+	AccessToken  string
+	RefreshToken string
+	AccessUUID   string
+	RefreshUUID  string
+	AtExpires    int64
+	RtExpires    int64
+}
+
 const (
-	N            = 1048576
-	r            = 8
-	p            = 1
-	keyLen       = 32
-	saltLenBytes = 16
```

Comment:
Why is it removed?

Reply:
It's not, just moved it down by several lines